src/components/tabs/TabsBar.js

```diff
@@ -1,7 +1,9 @@
+import touch from '~util/touch'
+
 export default {
   name: 'tabs-bar',
 
-  inject: ['isScrollable'],
+  inject: ['isScrollable', 'isMobile'],
```

Comment:
Yeah I know. In this particular instance it doesn't matter, because these functions are only used inside a computed prop which also depends on a variable that changes with resize (isOverflowing).